x-pack/test/functional/services/ml/api.ts

```diff
@@ -1591,6 +1591,16 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       }
     },
 
+    async startTrainedModelDeploymentES(deploymentId: string) {
```

Comment:
In the [route definition](https://github.com/elastic/kibana/blob/afac605a82859a72135d84ada7661d54ab5351b0/x-pack/platform/plugins/shared/ml/server/routes/trained_models.ts#L416) for `/internal/ml//trained_models/{modelId}/deployment/_start` the path param seems to be the `modelId` and `deploymentId` is an [optional query param](https://github.com/elastic/kibana/blob/afac605a82859a72135d84ada7661d54ab5351b0/x-pack/platform/plugins/shared/ml/server/routes/schemas/inference_schema.ts#L32).

I'm not sure whether the `/_ml/trained_models/${deploymentId}/deployment/_start` definition is different from the `/internal/ml//trained_models/{modelId}/deployment/_startl` definition.

If it's the same, can we call this `modelId` instead of `deploymentId` for clarity?

```suggestion
    async startTrainedModelDeploymentES(modelId: string) {
```